### PR TITLE
fix(detect): Detect _TZE200_wsbfwodu as AlecoAir HA-08

### PR DESCRIPTION
**Pull request by stefan-golinschi**
* Update tuya.ts Support for AlecoAir HA-08 Smart Thermostat Head

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -4488,6 +4489,7 @@ const definitions: DefinitionWithExtend[] = [
             {vendor: 'Unknown/id3.pl', model: 'GTZ08'},
             tuya.whitelabel('Moes', 'ZTRV-ZX-TV01-MS', 'Thermostat radiator valve', ['_TZE200_7yoranx2']),
             tuya.whitelabel('Moes', 'TV01-ZB', 'Thermostat radiator valve', ['_TZE200_e9ba97vf']),
+            {vendor: 'AlecoAir', model: 'HA-08 THERMO'},
```
Comment (Koenkk) [inline src/devices/tuya.ts:4492]:
```suggestion
            tuya.whitelabel('AlecoAir', 'HA-08_THERMO', 'Thermostat radiator valve', ['_TZE200_wsbfwodu']),
```

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `HA-08_THERMO.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by stefan-golinschi** [comment]
Hi Koenkk,
I just submitted the pull request with the product image.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**